start.sh

```diff
@@ -34,8 +34,25 @@ if [ $? -ne 0 ]; then
     exit
 fi
 
-echo Building asssets...
-webpack
+if [ $1 = "--setup-devenv" ] || [ $2 = "--setup-devenv" ]; then
+    echo Setup git modules...
+
+    git submodule init 
+    git submodule update
+    
+    echo Setup npm dependencies...
```

Comment:
Good idea, usually this file is called from Docker which is always guaranteed to start from the proper directory, but perhaps other people might want to use this script natively.